packages/at_repl/CHANGELOG.md

```diff
@@ -8,5 +8,4 @@
 
 ## 1.0.2
 
-- A more proper release. 
-
+- A more proper release. Fixed bugs, added more documentation.
```

Comment:
Reverse the order so that most recent is on top, and rewrite the changelog so that it follows [this format](https://github.com/atsign-foundation/at_libraries/blob/trunk/packages/at_commons/CHANGELOG.md)

Reply:
Should be fixed now.